### PR TITLE
Modified the names for PostgreSQL Flexible server.

### PR DESCRIPTION
**Pull request by pradeepkintali**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by pradeepkintali** [comment]
@qianwens can you please review this, we updated the models names not to conflict with existing stable version. This is for different version API and which is not announced yet.

File: specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/postgresql.json [hunk 1]
```diff
@@ -983,7 +983,7 @@
         "tags": [
           "Operations"
         ],
-        "operationId": "Operations_List",
+        "operationId": "FlexibleServerOperations_List",
```
Comment (QingqingYuan2) [inline specification/postgresql/resource-manager/Microsoft.DBforPostgreSQL/preview/2020-02-14-privatepreview/postgresql.json:986]: Let's revert back to Operations_List.

**Comment by QingqingYuan2** [comment]
@pradeepkintali , can you check the latest file for Mysql https://github.com/Azure/azure-rest-api-specs/blob/master/specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-privatepreview/mysql.json? Please have PostgreSQL align with that. For operation ids, all don't need to prefix with "flexibleServers". Let's revert back to that model. Also, please check with to add the readme.python.md to make sure they're generating into different namespaces.

**Comment by pradeepkintali** [comment]
> @pradeepkintali , can you check the latest file for Mysql https://github.com/Azure/azure-rest-api-specs/blob/master/specification/mysql/resource-manager/Microsoft.DBforMySQL/preview/2020-07-01-privatepreview/mysql.json? Please have PostgreSQL align with that. For operation ids, all don't need to prefix with "flexibleServers". Let's revert back to that model. Also, please check with to add the readme.python.md to make sure they're generating into different namespaces.

**Review by QingqingYuan2 — APPROVED** [review]

**Review by qianwens — APPROVED** [review]

**Pull request merged**